ko/services/camera.md

```diff
@@ -377,14 +377,16 @@ Other components like a GCS will typically only use the camera `BATTERY_STATUS.b
 
 | Command                                                                                                                                                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Thank you. Can you remove these changes in the ko and zh versions. The changes in english should round-trip to here through our translation service. If we do it in github there can be clashes that I have to resolve manually.